js/freewheel.js

```diff
@@ -540,6 +541,34 @@ OO.Ads.manager(function(_, $) {
       // if (fwContext && _.isFunction(fwContext.dispose)) fwContext.dispose();
     };
 
+    var onResize = function() {
+      //Overlay placement issue - PBI-1227 as of 12/9/2015
+      //The main issue with Freewheel is when notifying their SDK of video size changes,
+      //Freewheel only attempts to resize ads and not re-position ads.
+      //In the Freewheel SDK, overlay ads do not have a resize function and thus nothing happens.
+
+      //We want to force the renderer to reposition the overlay ads. The Freewheel SDK has functions
+      //that accomplish this, but are undocumented.
+      if(currentPlayingSlot){
+        //These two calls update Freewheel with the new video sizes. At this point
+        //Freewheel will attempt to resize any ads
+        fwContext.setContentVideoElement(amc.ui.ooyalaVideoElement[0]);
+        fwContext.registerVideoDisplayBase(amc.ui.adWrapper.attr("id"));
```

Comment:
That makes sense - but if those have to change, won't this have to change?  Basically there are several places where we have to call these two functions (and sometimes just one or the other) and i filtered it down to two basic concepts.  One sets up freewheel to properly render linear ads and the other sets it up to properly render nonlinear ads.  I think we should use those because if it ever changes there it will probably need to change here.  Based on my checking i think the commands and parameters are currently the same